src/Microsoft.ML.Data/DataLoadSave/CompositeDataLoader.cs

```diff
@@ -38,13 +39,15 @@ public CompositeDataLoader(IDataLoader<TSource> loader, TransformerChain<TLastTr
 
             Loader = loader;
             Transformer = transformerChain ?? new TransformerChain<TLastTransformer>();
+            _disposed = false;
```

Comment:
`bool`s are initialized to `false` by the runtime. There is no need to re-initialize them to `false` here.